docs/walkthroughs/local-setup.adoc

```diff
@@ -110,21 +110,132 @@ as part of your `ansible-playbook` command.
 For developing the OpenShift UI locally, see link:https://github.com/aerogear/mobile-core/blob/master/docs/walkthroughs/developing-ui-locally.adoc[this guide].
 
 [[creating-mobile-apps]]
+
+*Supported platforms/types*
+
+* Android
+* iOS
+* Cordova
+* Xamarin
+
 == Creating Mobile Apps
 
+[[openshift-ui]]
+=== Openshift UI
+
 Once the installer is complete you should be able to access OpenShift at
 https://192.168.37.1:8443/console/. You will need to accept the
 self-signed certificate.
 
-You can login using `developer` and any password.
+==== Create a new client
+
+Under the `mobile` tab select `apps` then choose a desired platform for your client app:
+
+image::images/new-client.png[new client]
+
+Fill the client app form with the proper information:
+
+image::images/new-client-1.png[new client form]
+
+You can check your recently client app status in your project now (provisioning might take a while):
+
+image::images/new-client-2.png[new client status]
+
+===== Retrieve an existing client
+
+You can check your client app info by going to `Applications > Provisioned Services` and selecting your client app:
+
+image::images/client-info.png[client info]
+
+image::images/client-info-1.png[client info defails]
+
+===== Delete an existing client
+
+You can delete your client app by clicking on `Actions > Delete` under the client app details:
+
+image::images/client-delete.png[delete client]
+
+You will also need to manually delete the client app secret (`Resources > Secrets`):
```

Comment:
When I tested, deleting the client app also deleted the client app secret as well.